public/styles.css

```
@@ -1,5 +1,5 @@
 @import url('./css/variables.css?v=15');
 @import url('./css/base.css?v=16');
 @import url('./css/crystal.css?v=12');
-@import url('./css/components.css?v=50');
+@import url('./css/components.css?v=51');
```

Stylelint (17.9.0)

[error] Expected "url('./css/components.css?v=51')" to be "'./css/components.css?v=51'" (import-notation)

(import-notation)